README.rst

```diff
@@ -342,11 +342,16 @@ Python 3 Support
 Some distributions support installing Salt to use Python 3 instead of Python 2. The availability of
 this offering, while limited, is as follows:
 
+- CentOS 6
```

Comment:
Please remove this and update the comments below. Py3 is not supported or tested on CentOS 6.

The `-x` comments in the script help are for Python 2.7 installations, not 3, since CentOS 6 ships with Py2.6.

Reply:
my bad, will do